### ISSUE TITLE
Feature: recursive "file" listing (-> pathana)

### ISSUE THREAD

**Issue by mrustl** [KWB-R/kwb.nextcloud#9]
Is it possible to do an recursive file listing. For example start the download from the "Budget" folder as the "budget" R script also requires data from both folders. May be also relevant for doing a "fakin" file list overview as required for pathana

```
Budget/
Budget/10_Filled_out_forms/....
Budget/20_Summary_Files/Partner_country_type.csv
```

**Comment by mrustl** [comment]
for the "budget" case. May be it is better to place the **Partner_country_type.csv** in the **10_Filled_out_forms* folder, so the  **20_Summary_Files** folder just contains data generated by R. What do you think @hsonne ?

**Comment by mrustl** [comment]
@hsonne, maybe this is quite simple by adding an header **Depth: <number-of-desired-levels>**(https://docs.nextcloud.com/server/latest/developer_manual/client_apis/WebDAV/basic.html#getting-properties-for-just-the-folder)

**Comment by mrustl** [comment]
ok. at least getting the whole tree with "Depth: infinity" is not possible (https://github.com/nextcloud/server/issues/5947)

**Comment by hsonne** [comment]
> 
> 
> for the "budget" case. May be it is better to place the **Partner_country_type.csv** in the **10_Filled_out_forms** folder, so the **20_Summary_Files** folder just contains data generated by R. What do you think @hsonne ?

This is related to kwb.budget and not kwb.nextcloud, right?

**Comment by hsonne** [comment]
Argument `recursive` available in `list_files()` since 05c4e8716edcd27c9c4e25153b1e21045b22d439
